### DISCUSSION TITLE
[fr] reproducible F-Droid app

### DISCUSSION THREAD

**Discussion by ghost** [plateaukao/einkbro#218] [Ideas]
if possible can you please app reproducible F-Droid app. for details see [Towards a reproducible F-Droid](https://f-droid.org/en/2023/01/15/towards-a-reproducible-fdroid.html).

jindam, vani

**Comment by plateaukao** [comment]
Hi, @jindam-vani 
I am not sure what it's really for.
If its purpose is just to make sure that the apk published on f-droid is the same as the one I provided (actually I didn't provide apks to f-droid, they build it from Github source codes), and you don't trust f-droid, you can simply download the latest apk I released on Github. Although it's a bit tedious to manually upgrade it, it's still a doable way. (I usually go to settings > About to upgrade EinkBro on all my devices, instead of using f-droid myself) 

If it's for other purposes, could you brief it here?
According to the url, it says:
>However, it was never widely used. One of the reasons is that it sounds difficult to achieve. We had few reproducible apps (to be precise: only 6) and some of them suffered from problems because of the use of reproducible builds. 

That does not encourage me to do so, and the tool link they provided are full of command-line tools. If you know how to do this task for EinkBro, you're welcome to send a PR, I would love to review it. If not, I would prefer spending time on adding new features, or polish EinkBro's UI.

**Reply by jindamvani** [reply]
@plateaukao  fyi: 
* [Total number of apps using RB: 6 + 75 = 81. Last updated: 2023-03-04](https://gitlab.com/obfusk/fdroid-misc-scripts/-/blob/master/reproducible/overview.md)
* [Encourage new apps enable reproducible build](https://gitlab.com/fdroid/fdroiddata/-/issues/2816)
